src/main/java/racingcar/Car.java

```diff
@@ -0,0 +1,37 @@
+package racingcar;
+
+import java.util.Random;
+
+public class Car {
+    private final String name;
+    private int position = 0;
+
+    public Car(final String name) {
+        if (name.length() > 5) {
+            throw new IllegalArgumentException();
+        }
+        this.name = name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void move() {
+        final var condition = new Random().nextInt(10);
+        move(condition);
+    }
+
+    public void move(final int condition) {
+        if (condition >= 4) {
+            position++;
+        }
+    }
+
+    public void move(final MovingStrategy movingStrategy) {
+        if (movingStrategy.movable()) {
+            position++;
+        }
+    }
+
+}
```

Comment:
파일의 마지막 라인에 개행을 하는것은 어떨까요?


```suggestion
    }
}

```

래퍼런스 : https://jojoldu.tistory.com/673